autospec/specfiles.py

```diff
@@ -1326,6 +1322,8 @@ def write_qmake_pattern(self):
             extra_qmake_args = "-spec linux-clang "
         if config.config_opts['use_lto']:
             extra_qmake_args += "-config ltcg "
+        else:
+            flags.extend(["-fno-lto"])
```

Comment:
Should this be ```extras_qmake_args += "-fno-lto " instead? (flags isn't a think in the write_qmake_pattern). Or does this need to exist at all since the arg isn't being passed? Maybe @thiagomacieira can give me a little insight here.